ee/tabby-ui/app/(dashboard)/settings/(integrations)/repository/[kind]/detail/components/gitlab-provider-detail.tsx

```diff
@@ -123,118 +129,239 @@ function toStatusBadge(status: RepositoryProviderStatus) {
   }
 }
 
-const LinkedRepoTable: React.FC<{
-  data: GitlabRepositories | undefined
+const ActiveRepoTable: React.FC<{
+  providerId: string
   providerStatus: RepositoryProviderStatus | undefined
   onDelete?: () => void
-}> = ({ data, onDelete, providerStatus }) => {
+}> = ({ onDelete, providerStatus, providerId }) => {
+  const [page, setPage] = React.useState(1)
+  const {
+    repositories: inactiveRepositories,
+    setRepositories: setInactiveRepositories,
+    isAllLoaded: isInactiveRepositoriesLoaded
+  } = useAllInactiveRepositories(providerId)
+
+  const doFetchRepositories = (
```

Comment:
Maybe the name can just be `fetchRepositories`?

Reply:
renamed